tweedie/tests/test_tweedie.py

```diff
@@ -127,6 +127,39 @@ def test_cdf_to_ppf(mu, p, phi):
     else:
         assert_allclose(qs, xs)
 
+# these tests basically verify that _logpdf and _logcdf broadcast the arguments
+# before calling the function that estimates the tweedie loglike and logcdf
+@mock.patch("tweedie.tweedie_dist.estimate_tweedie_loglike_series")
+def test_broadcasting_pdf(mock_estimate_tweedie):
```

Comment:
@thequackdaddy Does this work for you? It just checks that the estimate function are called with broadcast parameters when we pass just integers and not arrays. I mocked the calls to estimate and just check if the call was done properly.

Reply:
Seems good! I'll package this up and put on pypi shortly.

Reply:
Oh wait... this breaks older scipy. Would you look into that?

Reply:
Sorry about that, @thequackdaddy! We don't have access to older Python versions here at SF so it's hard to anticipate this kind of error. It seems to be somethin with how I used some of the mocking methods. I will take a look here. I'll make sure everything is passing before finalizing this.

Reply:
Ok, changed it a little and now it seems to be working. Just some changes on how mocking works between Python versions I guess.